docker-compose/readme.md

```diff
@@ -321,99 +333,106 @@ curl -k -v --header "Authorization: Bearer ${ACCESS_TOKEN}"  "${HOST}/v1/fhir/Ex
+   ```
+   docker-compose/run_integration_tests_local_keybase.sh dc-debug
+   ```
 
-     The currently checked out (or working branch) will be used.
+2. Using a Docker one-off run using the same image (bb2-cbc-build) as CBC. This takes longer, but provides a better test before using in CBC.
 
-     ```
-     docker-compose/run_integration_tests_local_keybase.sh cbc
-     ```
+   The currently checked out (or working branch) will be used.
 
-  3. Using the docker-compose local development setup and containers with local bfd as backend.
+   ```
+   docker-compose/run_integration_tests_local_keybase.sh cbc
+   ```
 
-     The currently checked out (or working branch) will be used.
+3. Using the docker-compose local development setup and containers with local bfd as backend.
 
-     ```
-     docker-compose/run_integration_tests_local_keybase.sh local
-     ```
-     To debug integration tests:
+   The currently checked out (or working branch) will be used.
 
-     ```
-     docker-compose/run_integration_tests_local_keybase.sh local-debug
-     ```
+   ```
+   docker-compose/run_integration_tests_local_keybase.sh local
+   ```
+
+   To debug integration tests:
+
+   ```
+   docker-compose/run_integration_tests_local_keybase.sh local-debug
+   ```
 
 NOTES:
-  * The settings variables in the `docker-compose/run_integration_tests_local_keybase.sh cbc` may need to be updated to match your local development platform.
-  * For the CBC related setup see these files for more details:
-    * `Jenkinsfiles/Jenkinsfile.cbc-run-integration-tests` - Jenkinsfile for running the tests in a CBC project/job. 
-    * `Jenkinsfiles/cbc-run-integration-tests.yaml` - Kubernetes docker container specification.  These settings will also need to be updated when there are CBC image naming changes.
+
+- The settings variables in the `docker-compose/run_integration_tests_local_keybase.sh cbc` may need to be updated to match your local development platform.
+- For the CBC related setup see these files for more details:
+  - `Jenkinsfiles/Jenkinsfile.cbc-run-integration-tests` - Jenkinsfile for running the tests in a CBC project/job.
+  - `Jenkinsfiles/cbc-run-integration-tests.yaml` - Kubernetes docker container specification. These settings will also need to be updated when there are CBC image naming changes.
 
 ## Developing and Running Selenium tests in Local Development
+
 You can run selenium tests by following below steps:
 
-  1. Make sure there is no blue button server and its dependent services running
-  2. Go to the base directory of the local repo and run:
+1. Make sure there is no blue button server and its dependent services running
+2. Go to the base directory of the local repo and run:
+
+   use MSLSX (default)
+
+   ```
+   ./docker-compose/run_selenium_tests_local_keybase.sh
+   ```
+
+   ```
+   ./docker-compose/run_selenium_tests_local_keybase.sh mslsx
+   ```
+
+   use SLSX
```

Comment:
What are all these other changes related to?

Reply:
thanks Brady for reviewing and comments, the readme.md has many diffs due to mark down auto formating, e..g the indentation used to be 4 space tab, the formatter makes it 2 space tab.